<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="bk_check">
         <property name="text">
          <string>버거킹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mom_check">
         <property name="text">
          <string>맘스터치</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="h_check">
         <property name="text">
          <string>홍루이젠</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="l_check">
         <property name="text">
          <string>롯데리아</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mc_check">
         <property name="text">
          <string>맥도날드</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="city1">
         <item>
          <property name="text">
           <string>전체검색</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>서울</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>부산</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>울산</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>인천</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>대전</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>광주</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>대구</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>경기</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>강원</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>충청북도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>충청남도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>경상북도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>경상남도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>전라북도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>전라남도</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>제주</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>세종</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="city2"/>
       </item>
       <item>
        <widget class="QComboBox" name="district"/>
       </item>
       <item>
        <widget class="QComboBox" name="street"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_search">
         <property name="text">
          <string>검색</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="result_count">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
